services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -407,4 +378,37 @@ private void setUpIndividualRequest(final IIndividualRequest request) {
         request.setBulkSubmission(bulkSubmission);
     }
 
+    @Test
+    public void setIndividualRequestDaoTest() throws Exception {
+
+        messageTaskService.setIndividualRequestDao(mockIndividualRequestDao);
```

Comment:
The messageTaskService object is initialised with the mockIndividualRequestDoa object in setUp().  To confirm that the set method works this test needs to set it to a different instance of an IndividualRequestDoa object.  Setting it to the same one used in setUp() means we can't determine if the set method worked or not.

This also applies to the setMessagingUtilityTest() and setGlobalParametersCacheTest() tests.

Reply:
Done